Program.cs

```diff
@@ -629,57 +654,93 @@ public void itemShop()
             if (hasPlayerBeenInShop == true)
             {
                 somethingCool();
+                if (playerHasWatch == false)
+                {
+                    GetWatch(totalXP);
+                }
                 shopCommonPlace();
             }
             else
             {   
-                TypeWriter.WriteLine($"Hello {cottonUserName} ");
                 shopCommonPlace();
                 hasPlayerBeenInShop = true;
             }
             
         }
 
-        private void shopCommonPlace()
+        private void GetWatch(int totalXP)
         {
-            TypeWriter.WriteLine(new Text("Are you looking for "),
-                                 new Text("(w)weapons, ", Colours.Attack, TypeWriter.Speed.Talk),
-                                 new Text("(a)armour ", Colours.Protection, TypeWriter.Speed.Talk),
-                                 new Text("or ", Colours.Speech, TypeWriter.Speed.Talk),
-                                 new Text("(m)medicine ", Colours.Health, TypeWriter.Speed.Talk),
-                                 new Text(":)", Colours.Speech, TypeWriter.Speed.Talk));
-            if (hasPlayerBeenInShop == true)
+            if (totalXP >= 300)
             {
-               if (playerHasSeenMorse == true)
-               {
-                   TypeWriter.WriteLine("(h) to hear it again or (l) to leave", TypeWriter.Speed.Talk);
-               }
-               else if (playerHasSeenMorse == false)
-               {
-                   TypeWriter.WriteLine("(h) to hear something cool or (l) to leave", TypeWriter.Speed.Talk);
-               }
+                TypeWriter.WriteLine($"Hey {cottonUserName} do you want to trade {priceForWatch} Gold for my pocket watch: yes/no", TypeWriter.Speed.Talk);
+                string Awnser = Console.ReadLine();
+                if (Awnser == "yes")
+                {
+                    TypeWriter.WriteLine();
+                    TypeWriter.WriteLine("Great you've got a pocket watch :)", TypeWriter.Speed.Talk);
+                    TypeWriter.WriteLine();
+                    Thread.Sleep(100);
```

Comment:
Why are you sleeping here... if it is because the screen will auto clear, then 100ms is not enough time for anyone to read that.

Reply:
the screen doesn't clear it's just for some effect but I changed to sleep for 200 to give the user more time